src/tests/functional/plugin/shared/include/subgraph_tests/split_concat_memory.hpp

```diff
@@ -5,18 +5,11 @@
 #pragma once
 
 #include "shared_test_classes/subgraph/split_concat_memory.hpp"
+#include "common_test_utils/data_utils.hpp"
 
-namespace SubgraphTestsDefinitions {
+namespace subgraph_tests_definitions {
```

Comment:
use the same namespace as for tests based on API2.0

Reply:
done